extensions/dacpac/src/test/wizardServiceInteraction.test.ts

```diff
@@ -56,6 +57,11 @@ describe('Dacfx wizard with connection', function (): void {
 
 	it('Should call all service methods correctly', async () => {
 		wizard.model.server = connectionProfileMock;
+		wizard.model.potentialDataLoss = true;
+		wizard.model.upgradeExisting = true;
+
+		const fileSizeStub = sinon.stub(utils, 'getFileSize');
+		fileSizeStub.resolves(TypeMoq.It.isAny());
```

Comment:
it's a little weird that the fileSizeStub is resolving TypeMoq.It.isAny. The actual function is supposed to return a number or undefined, so it would be more clear to return a number here.